src/openms/include/OpenMS/DATASTRUCTURES/DateTime.h

```diff
@@ -157,9 +168,21 @@ namespace OpenMS
     /// Returns the current date and time
     static DateTime now();
 
-    ///Sets the undefined date: 00/00/0000 00:00:00
+    /// Returns true if the date time is valid
+    bool isValid() const;
+
+    /// return true if the date and time is null 
+    bool isNull() const;
+
+    /// Sets the undefined date: 00/00/0000 00:00:00
     void clear();
 
+    /// Returns a string representation of date and time in the provided format
+    String toString(const std::string& format) const;
```

Comment:
can you add some examples of allowed formats?
Otherwise its a bit hard to guess what is valid here.